kmip/tests/unit/pie/test_client.py

```diff
@@ -334,6 +383,19 @@ def test_create_on_invalid_length(self):
 
     @mock.patch('kmip.pie.client.KMIPProxy',
                 mock.MagicMock(spec_set=KMIPProxy))
+    def test_create_on_invalid_cryptographic_usage_mask(self):
+        """
+        Test that a TypeError exception is raised when trying to create a
+        symmetric key with invalid cryptographic_usage_mask.
+        """
+        args = [enums.CryptographicAlgorithm.AES, 256]
+        kwargs = {'cryptographic_usage_mask':
+                  [enums.CryptographicUsageMask.ENCRYPT, 1]}
+        with ProxyKmipClient() as client:
+            self.assertRaises(TypeError, client.create, *args, **kwargs)
```

Comment:
Sure. But let me get clear. In this case, you want to raise an exception or not? Currently logic is that it will.